### ISSUE TITLE
Calling not existing method or attribute is ignored by interpreter without error

### ISSUE THREAD

**Issue by dvojtise** [gemoc/ale-lang#28]
in a simple ALE program (called as a GEMOC language)

if I have some incorrect instructions such as
```
self.whatever.foo();
self.foo();
```
(neither whatever, nor foo() exist)

The interpreter continues without any error/warning !?

**Comment by fcoulon** [comment]
Yes it's a bug. We should print error messages about calls to missing methods during the execution.

**Comment by dvojtise** [comment]
should be fixed by https://github.com/eclipse/gemoc-studio-execution-ale/commit/730648c6b91fb6d5409b1c348b535119c3981cce

**Comment by fcoulon** [comment]
Can I close the issue ?

**Comment by dvojtise** [comment]
This is better but several cases are not handled:

add the wrong statement in the `@init`method (the message is only "An error occured during evaluation of a query")

add the wrong statement in the `@main`method (no message at all)
